tests/test_crypto.py

```diff
@@ -3740,6 +3740,27 @@ def test_modification_pre_verify(self):
         store_ctx.set_store(store_good)
         self.assertEqual(store_ctx.verify_certificate(), None)
 
+    def test_verify_with_time(self):
+        """
+        :func:`verify_certificate` raises error when the verification time is
```

Comment:
I know the code base is inconsistent in this regard, but please don't use type prefixes like `:func:` in test doc strings.  They are never rendered and there’s no way to verify them at all.